#for dynamixel
R_THUMB_JOINT0:
  ID: 1
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
R_THUMB_JOINT1:
  ID: 2
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
R_INDEX_JOINT0:
  ID: 3
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
  Homing_Offset: 341
R_INDEX_JOINT1:
  ID: 4
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
  Homing_Offset: 341
R_MIDDLE_JOINT0:
  ID: 5
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
  Homing_Offset: 341
R_LOCK_JOINT0:
  ID: 6
  Return_Delay_Time: 0
  Operating_Mode: 5
  Current_Limit: 740
  Homing_Offset: 341
